Pipfile

```
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 ago = "~=0.0.95"
+black = "==23.7.0"
```

Is there any reason not to change this so we'll pick up minor updates?

```suggestion
black = "~=23.7.0"
```

---

> Is there any reason not to change this so we'll pick up minor updates?

I did not change this and do not see it when I pull down from GitHub or look at the code for the PR branch.  Don't know what is going on with GitHub (or my eyes).

---

Ah, this is just from having run `pipenv install` before - this suggestion adjusts the reference in the `Pipfile` to allow for minor or patch level updates, which Dependabot helps us manage.

---

Actually, let's not worry about this right now - the dependencies overall will be tackled soon enough as it is!